### ISSUE TITLE
Defining multiple jobs with epoch time expression behaves unexpectedly 

### ISSUE THREAD

**Issue by mctrp** [paragi/timexe#5]
Hi Simon,

Thank you for creating this nice piece of code!

I have a question about registering multiple timers using the `@<epoch>[.<faction of second>]` time expression as input.

Whenever I try to register multiple jobs to fire once at a specific epoch timestamp they don't seem to execute correctly. 

Am I overlooking something here perhaps?

**Environment:** node `v12.13.1` on windows 10

**Example code**
```js
var timexe = require('timexe');

var epochTarget = Math.floor(Date.now() / 1000) + 10; // set job 10 seconds from the start of the script

// register events
var event1 = timexe(`@${epochTarget}.000`, function(){console.log("hello world")});
var event2 = timexe(`@${epochTarget+5}.000`, function(){console.log("hello world + 5sec")});
var event3 = timexe(`@${epochTarget+10}.000`, function(){console.log("hello world + 10sec")});

console.log(event1);
console.log(event2);
console.log(event3);
```

**Expected output**
```
{ result: 'ok', error: '', id: 0 }
{ result: 'ok', error: '', id: 1 }
{ result: 'ok', error: '', id: 2 }
hello world
hello world + 5sec
hello world + 10sec
```

**Actual output**
```
{ result: 'ok', error: '', id: 0 }
{ result: 'ok', error: '', id: 1 }
{ result: 'ok', error: '', id: 2 }
hello world
hello world + 10sec
Schedule timer fired with invalid id: 2
hello world + 10sec
```

The commands did run at the correct time, but somehow not with the correct log message. With the `hello world + 10sec` showing up twice.

Did I overlook something?

Cheers,

Marcel

**Additional info**
(The `timexe.get()` command showed the following for this example:)

```
[
  {
    timex: '@1607776163.000',
    action: [Function],
    param: undefined,
    timeoutShortened: false,
    next: 1607776163,
    delay: 9009,
    now: 1607776153.991,
    timer: Timeout {
      _idleTimeout: 9009,
      _idlePrev: [TimersList],
      _idleNext: [TimersList],
      _idleStart: 32,
      _onTimeout: [Function],
      _timerArgs: [Array],
      _repeat: null,
      _destroyed: false,
      [Symbol(refed)]: true,
      [Symbol(asyncId)]: 2,
      [Symbol(triggerId)]: 1
    }
  },
  {
    timex: '@1607776168.000',
    action: [Function],
    param: undefined,
    timeoutShortened: false,
    next: 1607776168,
    delay: 14006,
    now: 1607776153.993,
    timer: Timeout {
      _idleTimeout: 14006,
      _idlePrev: [TimersList],
      _idleNext: [TimersList],
      _idleStart: 32,
      _onTimeout: [Function],
      _timerArgs: [Array],
      _repeat: null,
      _destroyed: false,
      [Symbol(refed)]: true,
      [Symbol(asyncId)]: 3,
      [Symbol(triggerId)]: 1
    }
  },
  {
    timex: '@1607776173.000',
    action: [Function],
    param: undefined,
    timeoutShortened: false,
    next: 1607776173,
    delay: 19006,
    now: 1607776153.993,
    timer: Timeout {
      _idleTimeout: 19006,
      _idlePrev: [TimersList],
      _idleNext: [TimersList],
      _idleStart: 32,
      _onTimeout: [Function],
      _timerArgs: [Array],
      _repeat: null,
      _destroyed: false,
      [Symbol(refed)]: true,
      [Symbol(asyncId)]: 4,
      [Symbol(triggerId)]: 1
    }
  }
]
```

**Comment by paragi** [comment]
Hi Marcel,
Thank you for taking your time to report this. 
I will look into it, as soon as possible, and get back to you.
Best regards, Simon

**Comment by paragi** [comment]
Sorry it took a while. It should be fixed now in version 1.0.3.
Thanks a lot!
